src/types/Status.ts

```diff
@@ -0,0 +1,5 @@
+export enum Status {
+  ALL = 'All',
+  ACTIVE = 'Active',
+  COMPLETED = 'Completed',
```

Comment:
enum's elements should be in camelCase:
```suggestion
  All = 'All',
  Active = 'Active',
  Completed = 'Completed',
```
please check other places